modules/arm_plugin/src/arm_config.cpp

```diff
@@ -61,8 +65,12 @@ InferenceEngine::Parameter Configuration::Get(const std::string& name) const {
         return {_ref};
     } else if (name == CONFIG_KEY_INTERNAL(LP_TRANSFORMS_MODE)) {
         return {_lpt};
+    } else if (name == CONFIG_KEY(ENFORCE_BF16)) {
+        return {_enforceBF16};
```

Comment:
does ARM plugin really support this option? I suppose it comes from Intel CPU plugin..

Reply:
ARM plugin doesn't support, but without this flag, behavior tests are failed, and same PERORMANCE_HINT

Reply:
maybe this is the problem of behavior tests, but not of the plugin?

Reply:
Our behavior tests don't use `ENFORCE_BF16` and `PERORMANCE_HINT` keys, and I don't know reasons but I can hardcode this keys (turn off ENFORCE_BF16) and set latency as performance hint

Reply:
but user will except that he can affect plugin behavior with such keys, but they don't mean anything for ARM..
it's a confusion for users